README.md

```diff
@@ -28,7 +28,7 @@ The recommended way to install the library is to use [composer](http://getcompos
 
 	{
 	  "require": {
-	    "backbone/chaching": "0.19.4"
+	    "backbone/chaching": "0.20.0"
```

Comment:
If you want to bump version, please change it everywhere it is used. Line 22 in README and then see constant `Chaching::VERSION`.